boards/arm/gd32e103v_eval/Kconfig

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2021 YuLong Yao <feilongphone@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_INIT_PRIORITY
+	int "Board initialization priority"
+	default 50
+	help
+	  Board initialization priority.
```

Comment:
this file is not required

Reply:
To be honest, I don't know what this config item means, but I saw it in both `gd32f450i_eval` and `gd32f403z_eval`.
After remove this file, it really compile and run success. So I'll remove this file.

Reply:
it was needed before pinctrl, not anymore ;-)